mungegithub/mungers/blunderbuss.go

```diff
@@ -90,7 +93,11 @@ func printChance(owners weightMap, total int64) {
 	}
 }
 
-func getPotentialOwners(author string, feats *features.Features, files []*githubapi.CommitFile, leafOnly bool) (weightMap, int64) {
+type ownerVetter interface {
```

Comment:
Feels bad to pass down a huge object and let `getPotentialOwners` even have visibility into so much state it has no reason to care about.